groups/groups.yaml

```diff
@@ -27,6 +27,11 @@ groups:
       - justinsb@google.com
       - stefan.schimanski@gmail.com
       - nikitaraghunath@gmail.com
+  - email-id: k8s-infra-cluster-operators-test@kubernetes.io
```

Comment:
@ameukam let's drop the `-test`. Also please add me `davanum@gmail.com` to help with setup/trouble shooting etc. Did you forget to add your id?

Reply:
I wanted to exclude myself from the testers. :-D

Reply:
please include me as well (spiffxp@google)

Reply:
@spiffxp @dims I included the changes requested.